### PR TITLE
Fix settings and display loader import

### PR DESCRIPTION
**Pull request by flohhhh**

### CODE REVIEW

File: src/ducks/balance/Balance.jsx [hunk 1]
```diff
@@ -345,8 +345,16 @@ class Balance extends PureComponent {
     const accounts = this.getAllAccounts()
     const triggers = triggersCollection.data
 
+    const hasNoAccounts =
+      accounts.filter(
+        a =>
+          a._id !== 'health_reimbursements' &&
+          a._id !== 'professional_reimbursements' &&
+          a._id !== 'others_reimbursements'
+      ) === 0
```
Comment (ptbrowne) [inline src/ducks/balance/Balance.jsx:354]: We do not have a `isVirtualAccount` helper function somewhere ?

File: src/ducks/balance/AccountsList.jsx [hunk 2]
```diff
@@ -43,9 +43,14 @@ export const AccountsList = props => {
     [filterByDoc, router]
   )
 
+  const sortedAndFilteredAccounts = useMemo(
+    () => getSortedAccounts(group, accounts).filter(a => a.status !== 'done'),
```
Comment (ptbrowne) [inline src/ducks/balance/AccountsList.jsx:47]: The `status` is an attribute of the `io.cozy.bank.accounts` ? I am a bit surprised by this.

File: src/ducks/balance/Balance.jsx (hunk 1 above)
Comment (flohhhh) [reply src/ducks/balance/Balance.jsx:354]: Not found, I created it.

Comment (ptbrowne) [reply src/ducks/balance/Balance.jsx:354]: I've checked quickly by grepping on "virtual" and found that normally virtual accounts should have a "virtual" property set to true.

File: src/ducks/balance/AccountsList.jsx (hunk 2 above)
Comment (flohhhh) [reply src/ducks/balance/AccountsList.jsx:47]: There is a status when importing accounts. Some accounts are in done, I guess they are accounts that have been added and then deleted.

File: package.json [hunk 3]
```diff
@@ -169,7 +169,7 @@
     "cozy-device-helper": "1.9.2",
     "cozy-doctypes": "1.69.0",
     "cozy-flags": "2.4.0",
-    "cozy-harvest-lib": "3.13.1",
+    "cozy-harvest-lib": "3.9.0",
```
Comment (ptbrowne) [inline package.json:172]: Pourquoi tu downgrade harvest ?

**Review by ptbrowne — APPROVED** [review]
LGTM

**Pull request merged**